src/components/Staking/IdoStakingForm.tsx

```diff
@@ -0,0 +1,252 @@
+import {
+  Box, Button, Heading, Image, Input, InputGroup, Spacer, Stack, Switch, Text
+} from '@chakra-ui/react'
+import { observer } from 'mobx-react-lite'
+import { Contract } from 'near-api-js/lib/contract'
+import { useStakingForm_Stake } from './StakingHooks'
+import StakingStatsStore from './StakingStatsStore'
+
+const darkPurple200 = '#2D225E'
+const darkPurple600 = '#DFD0FA'
+const darkPurple500 = '#9B8BD8'
+const darkPurple300 = '#402D82'
+const candy100 = '#D85988'
+const candy300 = '#FF61C0'
+const darkPurple0 = '#211A4A';
+const natural0 = '#EEEDF6'
+const natural200 = '#AFACC6'
+
+type Props = {
+  contract: Contract    // dev account
+  contractFT: Contract  // KULA token contract
+  contractIdo: Contract     // IDO contract
+  contractStaking: Contract // Staking contract
+  currentUser: any
+  nearConfig: any
+  walletConnection: any
+}
+
+export default observer(function StakingForm(props: Props) {
+
+  const {
+    frmStake_amount, set_frmStake_amount,
+    frmStake_lock_for, set_frmStake_lock_for,
+  } = useStakingForm_Stake(props, StakingStatsStore)
```

Comment:
Why u mix camelCase and snake case. 
its a question not a review

Reply:
This code is used in the old source, I have cloned it. When I start work on connect with the smart contract, I will update it to camelcase.

Reply:
Please also update these variable names to follow the standard naming convention.